### PR TITLE
Added description in skill for problem-solving and small talks category.

### PR DESCRIPTION
**Pull request by DravitLochan**
Partially Fixes issue #234
Added description in skill for problem-solving and small talks category.

### CODE REVIEW

**Comment by DravitLochan** [comment]
@madhavrathi @dynamitechetan @saurabhjn76 @Manveer695 @chiragw15 @betterclever @chashmeetsingh Please review!

**Review by dynamitechetan — APPROVED** [review]

**Review by madhavrathi — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]

**Review by chashmeetsingh — APPROVED** [review]

**Review by Manveer695 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please resolve conflicts and add images for each skill.

File: models/general/problemsolving/en/advanced_math.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #returns derivative of expression
 derivative *|differentiation of *
+!description : Ask for derivative of simple Mathematical functions
```
Comment (mariobehling) [inline models/general/problemsolving/en/advanced_math.txt]: Why is there a space in front of ":"? Please fix this in all files.

**Comment by mariobehling** [comment]
@DravitLochan Please finalize the version of this PR.

* Use the latest format and ensure all skills you are working on have a square image that is hosted on the repo

* Ensure you stick to the latest format
screenshot from 2017-08-03 08-01-25

**Review by dynamitechetan — COMMENTED** [review]
Please resolve conflicts

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please follow format and add all details.

**Comment by DravitLochan** [comment]
On to it.

**Comment by DravitLochan** [comment]
@mariobehling This PR is now ready to merge. @saurabhjn76 is already working to add images in problem-solving [here](https://github.com/fossasia/susi_skill_data/issues/242#issuecomment-318066623). Once this PR is merged, He may open one for images. Please review and merge. Thanks!

**Pull request merged**